sky/packages/sky/lib/widgets/framework.dart

```diff
@@ -1023,11 +1024,12 @@ abstract class MultiChildRenderObjectWrapper extends RenderObjectWrapper {
       walker(child);
   }
 
-  void insertChildRenderObject(RenderObjectWrapper child, dynamic slot) {
+  void insertChildRenderObject(RenderObjectWrapper child, Widget slot) {
     final renderObject = this.renderObject; // TODO(ianh): Remove this once the analyzer is cleverer
-    assert(slot == null || slot is RenderObject);
+    RenderObject nextSibling = slot is Widget ? slot.renderObject : null;
```

Comment:
slot is Widget  <--- The slot is always a widget according to the type signature of this function.